internal/tiltfile/tiltfile_state.go

```diff
@@ -168,6 +169,7 @@ func (s *tiltfileState) loadManifests(absFilename string, userConfigState model.
 		telemetry.NewExtension(),
 		updatesettings.NewExtension(),
 		encoding.NewExtension(),
+		extension.NewExtension(extension.NewGithubFetcher(), extension.NewLocalStore(filepath.Dir(absFilename))),
```

Comment:
really feeling the extension/extension ambiguity here

maybe we should name the package `tiltextension`?